content/topics/code-reviews/part-1-reviewers/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+content_type: topic
+ready: true
+flavours:
+- any_language
+prerequisites:
+  hard:
+  - why-and-how-to-review-code
+tags:
+- soft skills
+- communication
+title: Reviewer Communication During Code Reviews
+---
+
+The people who's code you review, are people. Remember this. Treat them with respect.
+
+- Lesson 1: [The Graceful Reviewer](https://www.youtube.com/watch?v=XY6eA2_2hOg)
```

Comment:
`The graceful reviewer` is better than Title Case. Title Case is even bad for titles. Just treat english as sentences